src/Symfony/Component/Validator/Constraints/AtLeastOneOfValidator.php

```diff
@@ -58,4 +65,20 @@ public function validate($value, Constraint $constraint)
             ->addViolation()
         ;
     }
+
+    private function createExecutionContext(): ExecutionContextInterface
+    {
+        return new ExecutionContext($this->context->getValidator(), $this->context->getRoot(), $this->getTranslator());
+    }
+
+    private function getTranslator(): TranslatorInterface
```

Comment:
This will create a new translator for every validator. Should we use a property to prevent creating a new instance every time?

Reply:
I reworked the patch. This is no longer needed.